### DISCUSSION TITLE
new jailmaker setup guide youtube

### DISCUSSION THREAD

**Discussion by mooglestiltzkin** [Jip-Hop/jailmaker#145] [General]
I saw stux posted a guide for youtube to setup jailmaker. Would it be possible to mention it in the doc?
https://www.youtube.com/watch?v=S0nTRvAHAP8

this would go well with the written guide for everyone especially the new comers.

**Comment by mrstux** [comment]
[![tempImageuepu80](https://github.com/Jip-Hop/jailmaker/assets/2467970/6afec755-dde7-4049-bf82-f7afdd39ea8a)](https://www.youtube.com/watch?v=S0nTRvAHAP8)

**Reply by Jip-Hop** [reply]
I have added it to the readme!
